Blish HUD/GameServices/DebugService.cs

```diff
@@ -161,23 +164,26 @@ public class DebugService:GameService {
 
         #region Debug Overlay
 
+        private ConcurrentDictionary<string, Func<GameTime, string>> _overlayTexts;
+        public  ConcurrentDictionary<string, Func<GameTime, string>> OverlayTexts => _overlayTexts;
```

Comment:
I think we'd want to have functions within the DebugService for adding and removing entries, if we want it to be publicly accessible.  If you don't want to go that far, I don't see any reason to keep the public member for now (since if we do add methods for adding/removing later, we'd need to deprecate this member anyways).

Reply:
I thought about it when implementing it, as I wanted to have extra functions too. In the end I just went with KISS, since all those functions would have been one liners anyway. Why would you want to remove direct access to the dictionary? Is there a reason why we should restrict getting the values from the dictionary?

How would you feel about a wrapper type? So that indexing still works and we can expose whatever we want.

Reply:
Ultimately I didn't want the backend implementation directly exposed in case we wanted to make changes later since those would be breaking.  If it's abstracted by a function and/or a wrapper type, we can modify it down the road without actually causing a breaking change.